pkg/domain/infra/tunnel/images.go

```diff
@@ -93,3 +94,12 @@ func (ir *ImageEngine) Pull(ctx context.Context, rawImage string, options entiti
 	}
 	return &entities.ImagePullReport{Images: pulledImages}, nil
 }
+
+func (ir *ImageEngine) Load(ctx context.Context, opts entities.ImageLoadOptions) (*entities.ImageLoadReport, error) {
+	f, err := os.Open(opts.Input)
```

Comment:
This is opening on the server not the client.

Reply:
It looks good to me. We're opening the file and pass it to the bindings which are throwing it over to the server. We're still on the client site.